.circleci/config.yml

```diff
@@ -312,6 +346,7 @@ jobs:
       - checkout
       - install_node_modules
       - install_ruby_modules
+      - setup_staging_secrets
```

Comment:
du coup si on utilise `CODEPUSH_KEY_ANDROID_STAGING` dans cette étape, tu peux le supprimer dans la commande "Deploy Android Code push for staging environment" plus bas